go.mod

```
@@ -5,12 +5,15 @@ go 1.23.0
 toolchain go1.24.1
 
 require (
+	github.com/defenseunicorns/go-oscal v0.6.2
```

```suggestion
	github.com/defenseunicorns/go-oscal v0.6.3
```

See https://github.com/defenseunicorns/go-oscal/releases/tag/v0.6.3